ui-admin/src/study/families/FamilyRelations.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import {
+  EnrolleeRelation,
+  Family
+} from '@juniper/ui-core'
+import { StudyEnvContextT } from 'study/StudyEnvironmentRouter'
+import {
+  ColumnDef,
+  getCoreRowModel,
+  getSortedRowModel,
+  useReactTable
+} from '@tanstack/react-table'
+import { basicTableLayout } from 'util/tableUtils'
+import Creatable from 'react-select/creatable'
+import { EnrolleeLink } from 'study/participants/enrolleeView/EnrolleeLink'
+
+/**
+ * Editable view of the relationships within a family.
+ */
+export const FamilyRelations = ({
+  family, studyEnvContext
+}:{
+  family: Family, studyEnvContext: StudyEnvContextT
+}) => {
+  const columns: ColumnDef<EnrolleeRelation>[] = [{
+    header: 'Enrollee',
+    accessorKey: 'enrolleeId',
+    cell: ({ row }) => {
+      const enrollee = family.members?.find(m => m.id === row.original.enrolleeId)
+      return <EnrolleeLink studyEnvContext={studyEnvContext} enrollee={enrollee}/>
+    }
+  }, {
+    header: 'Relation',
+    accessorKey: 'familyRelationship',
+    cell: ({ row }) => {
+      const relationship = row.original.familyRelationship
+      return <span className="fst-italic">
+        is the <Creatable
+          isDisabled
+          options={[]}
+          value={{ value: relationship, label: relationship }}
+          className={'d-inline-block'}
+        /> of
+      </span>
+    }
+  }, {
+    header: 'Target',
+    accessorKey: 'targetEnrolleeId',
+    cell: ({ row }) => {
+      const enrollee = family.members?.find(m => m.id === row.original.targetEnrolleeId)
+      return <EnrolleeLink studyEnvContext={studyEnvContext} enrollee={enrollee}/>
+    }
+  }]
+
+
+  const table = useReactTable({
+    data: family.relations || [],
+    columns,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel()
+  })
+  return <div>
+    {basicTableLayout(table)}
```

Comment:
the display of the table is a bit wonky due to the select sizing.  let's update BasicTableConfig to take a `trClass` and then you can pass `align-middle` to get everything centered

Reply:
Hmm, I'm not sure what you mean that it looks wonky. On my screen, everything is already centered and `align-middle` has no effect:
<img width="532" alt="image" src="https://github.com/broadinstitute/juniper/assets/17440010/61dcc792-9b49-4ed1-bbae-bad01209bd35">